utils/jwtUtil.js

```diff
@@ -197,7 +207,8 @@ function verifyBotToken(payload, req, cb) {
     assignHeaderValues(req, payload);
     return cb ? cb() : undefined;
   })
-  .catch(() => {
+  .catch((err) => {
+    console.log(err);
```

Comment:
did you intend to leave this in?

Reply:
Nope I did not! Thanks for pointing out!